tfx/experimental/templates/taxi/e2e_tests/test_utils.py

```diff
@@ -150,7 +169,7 @@ def _uncommentMultiLineVariables(self, filepath: Text,
     io_utils.write_string_file(path, ''.join(result))
     return path
 
-  def _copyTemplate(self):
+  def _copyTemplate(self):  # pylint:disable=invalid-name
```

Comment:
Thank you for taking care of this! Would you mind changing these names into "valid" style? Or just let them as is. Supressing lint errors when it is not really needed doesn't like a good idea...